docs/variants/qemu_q35/building-manual.md

```diff
@@ -31,13 +31,13 @@ Obtain Dasharo source code:
 Clone Dasharo EDK II fork:
 
 ```bash
-git clone https://github.com/Dasharo/edk2.git -b REVISION
+git clone https://github.com/Dasharo/coreboot.git -b REVISION
```

Comment:
Maybe add `--depth 1`, building doesn't require any more than that.  `--filter=blob:none` should be even better but I think it's a recent addition.
```suggestion
git clone https://github.com/Dasharo/coreboot.git -b REVISION --depth 1
```

Reply:
https://github.com/Dasharo/docs/compare/2a47bdd5f3eae81d35cc9545010489747bac76c6..fd1fc06fa12f4d9dde6a66b01a567657c6bb1a5d